taglib/src/main/java/com/reevoo/taglib/AbstractReevooMarkClientTag.java

```diff
@@ -119,7 +107,20 @@ protected void setClient(ReevooMarkClient client) {
             // with the parameter "reviews" instead of the parameter "per_page".
             queryStringParams.put("reviews",this.numberOfReviews);
         }
+        if (this.dynamicAttrs != null) {
+          for (Map.Entry<String,String> entry : this.dynamicAttrs.entrySet()) {
```

Comment:
We can put the result of calling this.dynamicAttrs.entrySet() in a variable to avoid the call to be executed at each loop. Or does the compiler already applies optimizations to this kind of things?